src/docs/release/breaking-changes/scrollable_alert_dialog.md

```diff
@@ -113,27 +109,43 @@ flutter:          └─SemanticsNode#32 <-- contents
 flutter:              label: "Huge content"
 ```
 
-2. This change may result in layout changes because of the scroll view.
+### This change may result in layout changes because of the scroll view.
 
 If the dialog was already overflowing, this change would correct the problem
 and this layout change is expected.
 
 A nested SingleChildScrollView in `AlertDialog.content` seems to still work
 properly if left in, but it should probably be removed if unintended since
-it might cause confusion.
-
-### Two-phase update:
+it might cause confusion:
 
-This change will be conducted in two phases.
-
-1. The first phase introduces a new parameter to AlertDialog called
-`scrollable`. It will default to `false`, so this new parameter will need
-to be set to `true` to reflect the latest changes. Any expected breaking tests
-described above should be fixed.
+Original code:
+```dart
+AlertDialog(
+  title: Text(
+    'Very, very large title that is also scrollable',
+    textScaleFactor: 5,
+  ),
+  content: SingleChildScrollView( // will not be scrollable
+    child: Text('Scrollable content', textScaleFactor: 5),
+  ),
+  actions: <Widget>[
+    FlatButton(child: Text('Button 1'), onPressed: () {}),
+    FlatButton(child: Text('Button 2'), onPressed: () {}),
+  ],
+)
+```
 
-2. The second phase would remove the `scrollable` parameter and have the
-AlertDialog widget be scrollable by default. This will require all AlertDialogs
-to be modified and have their `scrollable` parameters removed.
+Migrate to:
+```dart
+AlertDialog(
+  title: Text('Very, very large title', textScaleFactor: 5),
+  content: Text('Very, very large content', textScaleFactor: 5),
+  actions: <Widget>[
+    FlatButton(child: Text('Button 1'), onPressed: () {}),
+    FlatButton(child: Text('Button 2'), onPressed: () {}),
+  ],
+)
+```
 
 ## Timeline
```

Comment:
I did still have a question about the `Timeline` section -- What should this really contain? Or is this what I have already intended?

Reply:
Did you see Ian's PR? He fleshed this out a bit.

Reply:
I just saw it! In that sense, I'm not far along enough with my migration (have not implemented the hard breaking change) that it applies yet because I made a mistake with understanding the steps in the process. 

Could I add a placeholder in here for now and come back and add the correct version to the timeline once the hard breaking change has landed?

Reply:
yes